src/appengine/handlers/cron/fuzzer_weights.py

```diff
@@ -198,7 +198,7 @@ def _coverage_formatter(query_format, dataset):
 """
 
 COVERAGE_UNCHANGED_SPECIFICATION = QuerySpecification(
-    adjusted_weight=0.5,
+    adjusted_weight=0.65,
```

Comment:
Can you explain this more.

Reply:
This seemed less severe than some of the other issues, especially now that the thresholds have been raised a bit. I was surprised by how many pairings were actually matching this rule, and how heavily it was skewing the weights down.